tests/suites/test_suite_psa_crypto.function

```diff
@@ -3066,6 +3066,7 @@ void mac_sign( int key_type_arg,
                               sizeof( actual_mac ) - mac_length ) );
 
 exit:
+    psa_mac_abort( &operation );
```

Comment:
It seems there is the same kind of thing in `test_suite_psa_crypto_hash.function`. While at it, probably better to do it there as well.